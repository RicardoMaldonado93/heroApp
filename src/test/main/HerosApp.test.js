import React from "react";
import { mount } from "enzyme";
import { HerosApp } from "../../HerosApp";

describe("Tests for HerosApp", () => {

  test("should show correctly", () => {
    const wrapper = mount(<HerosApp />);
    expect(wrapper).toMatchSnapshot();
  });

});
