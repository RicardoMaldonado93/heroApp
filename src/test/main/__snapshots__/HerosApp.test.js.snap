// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for HerosApp should show correctly 1`] = `
<div>
  <div
    className="container mt-5"
  >
    <h1>
      Login
    </h1>
    <hr />
    <button
      className="btn btn-outline-primary"
      onClick={[Function]}
    >
      Sign In
    </button>
  </div>
</div>
`;
